src/helix-ui/elements/HXFileTileElement.less

```diff
@@ -98,33 +97,35 @@
   }
 
   #hxLink {
-    color: @gray-900;
```

Comment:
Removed because `@gray-900` is the default color for the File Tile, so we let inheritance take over.